src/tools/decomp.py

```diff
@@ -53,47 +99,284 @@
 
 import math
 import time
+from loadbal import qbicity, changeIndex, halfDecomp, addHsymmetry, adaptNeiList,reDistCellsHom,nodeGridSizeCheck
 
-# need to factorize n to find optimum nodeGrid
-def nodeGrid(n):
-	ijkmax = 3*n*n + 1
-	d1 = 1
-	d2 = 1
-	d3 = 1
-	for i in xrange(1,n+1):
-		for j in xrange(i,n+1):
-			for k in xrange(j,n+1):
-				if (i*j*k == n) and (i*i + j*j + k*k < ijkmax):
-					d1 = k
-					d2 = j
-					d3 = i
-					ijkmax = i*i + j*j + k*k
-	return Int3D(d1,d2,d3)
+__author__ = 'Horacio V Guzman'
+__email__ = 'horacio.v.g at gmail dot com'
+__version__ = '1.0'
```

Comment:
Why should a single Python have the different version from the whole espp package?

Reply:
The same was done for the PMI module, in this case for HeSpaDDA

Reply:
But [PMI](https://pypi.python.org/pypi/pmi) can be used outside of espresso++!

Reply:
Good Idea! it can run on its one(outside E++)... I am working on the package